pom.xml

```diff
@@ -990,6 +990,22 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>tycho-gpg-plugin</artifactId>
+				<version>${tycho-version}</version>
+				<executions>
+					<execution>
+						<id>pgpsigner</id>
+						<goals>
+						<goal>sign-p2-artifacts</goal>
+						</goals>
+						<configuration>
+							<keyname>4F23165B6AC51B15</keyname>
```

Comment:
Will this magic work for other Eclipse projects as well or this key is individual for every project @jonahgraham ?

Reply:
It is specific to CDT's key. See https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/3070